Minor updates

phpsa opened the pull request:
moved zindex up on editor in order to make the x work on image within an a tag (per the logo image on the template)
added a destroy method --- used if wnating to switch between more than one template.
html = html.replace(/dtf.\* /g, ""); on export was breaking the basic template.

phpsa committed:
Fixed css issue with cancel button and html cleanup issue

Commit:
Merge pull request #1 from crunchmail/master

update to latest

Commit:
Merge pull request #2 from crunchmail/master

prevent duplication block in block catalog

phpsa committed:
Minor js fixes

phpsa committed:
Merge branch 'master' of https://github.com/phpsa/dentifrice

phpsa committed:
Editor css fixes
added basic ability to have more than one template in local storage
added template preview image
added block to basic template
fixed spelling mistake in basic template configuration file

courgette commented:
Thanks for your contribution ! 

The good way is to modify files into src folder, gulp and then push both edited files : the ones you edit, and the ones gulp generate.
It's better to modify files into src folder, gulp and push them.
One more thing, can you make a Pull request by feature please. 

* 1 Pull Request for css fixes
* 1 Pull Request for preview img block
* 1 Pull Request for draft template (By the way, Can you explain me this feature ?)

courgette commented:
Thanks for your contribution ! 

You have modified files on dist folder, this folder is generated by gulp when you use gulp command line and used by bower package. 
The good way is to modify files into src folder, gulp and then push both edited files : the ones you edit, and the ones gulp generate.
It's better to modify files into src folder, gulp and push them.
One more thing, can you make a Pull request by feature please. 

* 1 Pull Request for css fixes
* 1 Pull Request for preview img block
* 1 Pull Request for draft template (By the way, Can you explain me this feature ?)

courgette commented:
Thanks for your contribution ! 

You have modified files on dist folder, this folder is generated by gulp when you use gulp command line and used by bower package. 
The good way is to modify files into src folder, gulp and then push both edited files : the ones you edit, and the ones gulp generate.
One more thing, can you make a Pull request by feature please. 
- 1 Pull Request for css fixes
- 1 Pull Request for preview img block
- 1 Pull Request for draft template (By the way, Can you explain me this feature ?)

phpsa commented:
Have been seeing how well this would intergrate with my internal systems - as i have some really noob users who want to create newsletters - they are loving it already even with the limited features as such.
some of what I have been looking at is a screen with a choice of muliple template designs that can be implemented - (or though even if you use multiple pages the same issue arises) - 

in the tickets i have placed some modified code options for you to look at and perhaps clean up better:

1) Updated the draft save and restore to actually include the template in use - that way it does not try insert the wrong html body into the wrong contianer - which caused a few issues with some of the temlplate designs i was playing with.

2) destroy method - resets to default to allow different template load and ininitailaisation without a page reload.

3) Updated logic to allow a block to be split into definitions - so for instance a hero block could have a heading and paragraph wiht the heading locked to the minimal controls in the ckeditor and the paragraph a higher level of controls....

All these tickets are in your ticketing system ....
